### PR TITLE
Fix build error with shlwapi function

### PR DESCRIPTION
**Pull request by zheqiMicrosoft**
Really sorry for making this mistake. (I forgot to explicitly disable std::filesystem when trying to build) I just realized that there are several other places I have to use the A version explicitly.

### CODE REVIEW

**Comment by rpavlik** [comment]
Can't rebase this automatically on master - can you do that (or just cherry-pick the one commit on top of master), or would you prefer that I do?

**Comment by zheqiMicrosoft** [comment]
I will do that. Thanks.

Best,
Kevin

________________________________
发件人: Ryan A. Pavlik <notifications@github.com>
发送时间: Monday, June 17, 2019 2:30:06 PM
收件人: KhronosGroup/OpenXR-SDK
抄送: Kevin Qin; Author
主题: Re: [KhronosGroup/OpenXR-SDK] Fix build error with shlwapi function (#60)


Can't rebase this automatically on master - can you do that (or just cherry-pick the one commit on top of master), or would you prefer that I do?

―

**Comment by zheqiMicrosoft** [comment]
I just rebased the PR. Can you double check if that works please? Thanks!

Best,
Kevin

________________________________
发件人: Ryan A. Pavlik <notifications@github.com>
发送时间: Monday, June 17, 2019 2:30:06 PM
收件人: KhronosGroup/OpenXR-SDK
抄送: Kevin Qin; Author
主题: Re: [KhronosGroup/OpenXR-SDK] Fix build error with shlwapi function (#60)


Can't rebase this automatically on master - can you do that (or just cherry-pick the one commit on top of master), or would you prefer that I do?

**Comment by rpavlik** [comment]
Looks reasonable to me, will let @yl-msft decide if it's good to merge, given the related work, apparently already coordinated.

**Comment by zheqiMicrosoft** [comment]
Thanks!

File: src/common/filesystem_utils.cpp [hunk 1]
```diff
@@ -315,14 +315,14 @@ bool FileSysUtilsParsePathList(std::string& path_list, std::vector<std::string>&
 
 bool FileSysUtilsFindFilesInPath(const std::string& path, std::vector<std::string>& files) {
     try {
-        WIN32_FIND_DATA file_data;
-        HANDLE file_handle = FindFirstFileA(path.c_str(), &file_data);
+        LPWIN32_FIND_DATAA file_data;
```
Comment (yl-msft) [inline src/common/filesystem_utils.cpp]: This is an output struct.  If you left this as a random pointer, the Find... functions below won't work correctly (or writing into random memory)

**Review by yl-msft — CHANGES_REQUESTED** [review]

File: src/common/filesystem_utils.cpp (hunk 1 above)
Comment (zheqiMicrosoft) [reply src/common/filesystem_utils.cpp]: I see. Thanks! I changed the file_data to WIN32_FIND_DATAA

**Review by yl-msft — APPROVED** [review]

**Pull request merged**